package.json

```diff
@@ -21,7 +21,8 @@
     "test:backstop-approve": "gulp backstop:approve",
     "docs:generate-decision-log-listing": "adr-log -d ./docs/adr -i ./docs/adr/index.md -e template.md",
     "audit": "better-npm-audit audit",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "node lib/check-govuk-and-hmrc-frontend-versions.js"
```

Comment:
we might need to copy this into the package, so it's available when installing - or is lib being included?

the last version we had it at top level with a gulp target to copy it into package

https://github.com/hmrc/hmrc-frontend/commit/2b846b5bde540fa463d69098a64d9402fb4e2acb#diff-25789e3ba4c2adf4a68996260eb693a441b4a834c38b76167a120f0b51b969f7L29

what do you think about moving it to the top level and making it "check-compatibility-with-govuk-frontend-version.js"

Reply:
or maybe, because I love verbose stuff "warn-about-potential-incompatibility-with-govuk-frontend-version.js" xD dunno